rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -87,6 +87,7 @@ docker run --name $docker_name \
   --volume /tmp/pai-root/alive/$APP_ID:/alive \
   --label PAI_HOSTNAME="$(hostname)" \
   --label PAI_JOB_NAME={{{ jobData.jobName }}} \
+  --label PAI_JOB_VC_NAME={{{ jobData.virtualCluster }}} \
```

Comment:
`PAI_VC_NAME` is sufficient?

Reply:
I think here means this job belong to which virtualCluster, maybe PAI_JOB_VC_NAME will be more precise?

Reply:
Ok, it's more precise.